packages/terra-consumer-layout/tests/nightwatch/DexH@CTheme.scss

```diff
@@ -15,4 +15,6 @@
   --terra-consumer-body-background-color: rgb(155, 218, 241);
   --terra-consumer-nav-background-color: rgba(255, 255, 255, 0.3);
   --terra-consumer-nav-active-link-background: #9BDAF1 url('https://d320j1w59ws5w0.cloudfront.net/3Q6UT7T_clinic-background.jpg');
+
+  --terra-consumer-font-family: 'Roboto', sans-serif;
```

Comment:
you should set it to something else, and have your solution designer or ux designer verify things are working correctly. This here doesn't really do much since you are just setting it back to the default